version.txt

```diff
@@ -1 +1 @@
-0.7.1
+0.7.2
```

Comment:
Would do this in a different PR. we also need to possible change SAVEDIR etc

Reply:
Done.